projects/plugins/jetpack/modules/publicize/ui.php

```diff
@@ -538,9 +568,10 @@ class="publicize-external-link"
 			 *
 			 * @param string $publicize_form Publicize Details form appearing above Publish button in the editor.
 			 */
-			echo apply_filters( 'publicize_form', $publicize_form );
+			echo apply_filters( 'publicize_form', $publicize_form ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Is that safe to ignore as we've escaped and sanitized the individual fields? Having said that this file isn't as important, as it's not used currently I don't think.

Reply:
As far as I can tell all the individual fields added above it are already escaped. Is there a better way to escape this, especially since we're allowing users to add their own code with the filter. My first thought would be `wp_kses` with a long list of allowed HTML but perhaps there's a better approach.

Are you sure it's unused? I thought this metabox was rendered when the Classic Editor is active.

Reply:
Ah yes, I thought this was the UI in the admin rather than the post. I wonder if this is actually being used anywhere. If we're not using it (and we can check some other places, like Yoast, WooCommerce, etc.) then maybe we can deprecate the filter.

It should be safe to ignore here. Perhaps we should add a comment as to why we we're doing that?